tools/const_finder/lib/const_finder.dart

```diff
@@ -64,12 +59,13 @@ class _ConstVisitor extends RecursiveVisitor<void> {
 
   @override
   void visitConstructorInvocation(ConstructorInvocation node) {
-    final Class parentClass = node.target.parent as Class;
+    final Class parentClass = node.target.parent! as Class;
     if (_matches(parentClass)) {
+      assert(node.location != null);
       nonConstantLocations.add(<String, dynamic>{
-        'file': node.location.file.toString(),
-        'line': node.location.line,
-        'column': node.location.column,
+        'file': node.location!.file.toString(),
+        'line': node.location!.line,
+        'column': node.location!.column,
```

Comment:
Consider caching `node.location!` in a local variable. This also makes assertion at L64 redundant.

Reply:
Done.